Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/ClusterDetail.java

```diff
@@ -490,4 +490,10 @@ public SparkSubmitStorageTypeOptionsForCluster getStorageOptionsType() {
             return SparkSubmitStorageTypeOptionsForCluster.ClusterWithUnknown;
         }
     }
+
+    @Override
+    public boolean isMFSEspCluster(){
```

Comment:
It might be better to create a new class `MfaHdiCluster` to extend `ClusterDetail`